torch/csrc/jit/serialization/pickler.h

```diff
@@ -296,8 +297,59 @@ uint64_t getStorageKey(const at::Tensor& tensor);
 // otherwise return false
 bool checkHasValidSetGetState(const std::shared_ptr<c10::ClassType>& cls);
 
-// Return a map of Tensor Metadata for serialization.
-// For now, it only takes care of `conj` and `neg` bit.
+// Declare BackendMeta serialization and deserialization function pointer types.
+using BackendMetaPtr =
+    void (*)(const at::Tensor&, std::unordered_map<std::string, bool>&);
```

Comment:
Nit (yes, it adds a bit of an overhead for C functions but allows one to bind some context if necessary)
```suggestion
    std::function<void(const at::Tensor&, std::unordered_map<std::string, bool>&)>;
```